### PR TITLE
Add Chroma Feedback to Notifications

### PR DESCRIPTION
**Pull request by redaxmedia**
# Description
Added Chroma Feedback to the notifications page... I would prefer a third party tooling page but this is a temporary solution suggested by Tyler in https://github.com/circleci/circleci-docs/issues/4209

# Reasons
Because it is awesome:
chroma-feedback

### CODE REVIEW

File: jekyll/_cci2/notifications.md [hunk 1]
```diff
@@ -149,4 +149,8 @@ workflows:
       - build
 ```
 
+## Notification with RGB hardware
```
Comment (teesloane) [inline jekyll/_cci2/notifications.md]:
```suggestion
## Third Party Notification Tools
```

File: jekyll/_cci2/notifications.md [hunk 2]
```diff
@@ -149,4 +149,8 @@ workflows:
...
 
+[Chroma Feedback](https://github.com/redaxmedia/chroma-feedback) is a command line tool in Python to turn your RGB powered hardware into an build indicator. The idea of such extreme visibility is to encourage developers to instantly repair their builds.
+
+![Terminal Session](https://cdn.rawgit.com/redaxmedia/media/master/chroma-feedback/terminal-session.svg)
```
Comment (teesloane) [inline jekyll/_cci2/notifications.md]: Can you remove this please? We try to reduce gif usage in our docs. I think having a link to the project should suffice?

Comment (redaxmedia) [reply jekyll/_cci2/notifications.md]: No problem...

File: jekyll/_cci2/notifications.md (hunk 1 above)
Comment (redaxmedia) [reply jekyll/_cci2/notifications.md]: Renamed it to Third Party Tools with subhead Chroma Feedback

**Comment by teesloane** [comment]
Merged! Thanks for contributing @redaxmedia !

**Comment by redaxmedia** [comment]
@teesloane Thank you very much aswell. I hope someone will respond to my request for the guest writer program?

**Pull request merged**